src/functions.php

```diff
@@ -110,6 +110,22 @@ function is_first_key_operator($document)
     return (isset($firstKey[0]) && $firstKey[0] === '$');
 }
 
+/**
+ * Returns whether we are currently in a transaction
+ *
+ * @internal
+ * @param array  $options  Command options
```

Comment:
Extra whitespace can be trimmed here since there's no other `@param` elements to align with. #pedantic

Reply:
I was wondering if you'd notice 😂